crates/hive-utils/scripts/start.sh

```diff
@@ -8,6 +8,7 @@ export PROXY_ACCOUNT_CLASS_HASH=$(starkli class-hash ${MADARA_PATH}/cairo-contra
     --rpc-external \
     --rpc-methods=unsafe \
     --rpc-cors=all \
+    --disable-madara-configs \
     --tmp \
```

Comment:
why are u setting a tmp path for the db? since it's a docker container, u can just let it go to the default madara-path